packages/docusaurus-plugin-vercel-analytics/src/options.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import {Joi} from '@docusaurus/utils-validation';
+import type {OptionValidationContext} from '@docusaurus/types';
+
+export type PluginOptions = {
+  id: string;
+  mode: 'auto' | 'production' | 'development';
+  debug: boolean;
+};
+
+export type Options = Partial<PluginOptions>;
+
+export const DEFAULT_OPTIONS: Partial<PluginOptions> = {
+  mode: 'production',
+  debug: false,
```

Comment:
Also use `undefined`?

https://vercel.com/docs/analytics/package#debug

> This option is automatically enabled if the NODE_ENV environment variable is available and either development or test.

Similarly I think using `docusaurus build --dev` should turn this plugin to debug mode

Reply:
Same, didn't understand the use of undefined however modified according to your comment